src/packages/end-to-end/package.json

```diff
@@ -6,12 +6,15 @@
 	"scripts": {
 		"import-database-sqlite": ". ./scripts/import-sqlite.sh",
 		"import-database-postgres": ". ./scripts/import-postgres.sh",
+		"import-database-mysql": ". ./scripts/import-mysql.sh",
 		"seed-postgres": "node ./scripts/seed-postgres.js",
 		"start-server": "cd app && pnpm start",
 		"dev-sqlite": "pnpm import-database-sqlite && pnpm start-server",
 		"dev-postgres": "createdb gw && pnpm seed-postgres && pnpm import-database-postgres  && pnpm start-server",
+		"dev-mysql": "export DATABASE_USERNAME=root && export DATABASE_PASSWORD=root && export DATABASE_HOST=localhost && pnpm import-database-mysql && pnpm start-server",
```

Comment:
What happens here if my local mysql has a different password?

Reply:
We probably want to make all these configurable on the users local.

Reply:
Done. Updated ReadMe to set the environment variables.